jib-core/src/main/java/com/google/cloud/tools/jib/docker/json/DockerManifestEntryTemplate.java

```diff
@@ -48,15 +48,15 @@
 public class DockerManifestEntryTemplate implements JsonTemplate {
 
   private String config = "config.json";
-  private List<String> repoTags = Collections.singletonList(null);
+  private final List<String> repoTags = new ArrayList<>();
   private final List<String> layers = new ArrayList<>();
 
   public void setConfig(String config) {
     this.config = config;
   }
 
-  public void setRepoTags(String repoTags) {
-    this.repoTags = Collections.singletonList(repoTags);
+  public void addRepoTag(String repoTag) {
```

Comment:
Oh, sorry I missed this part. Isn't `setRepoTags` required for deserialization? Do we deserialize a Docker manifest somewhere, possibly for the Docker base image feature? We should check if tags are properly populated in that case?

Reply:
Hopefully jackson can smartly make use of `addRepoTag`, but I think we need to confirm (if we are deserializing this field to make use of it).

Reply:
No, I don't think getters and setters affect de/serialization if we use `JsonTemplate`. Only fields.

```
@JsonInclude(JsonInclude.Include.NON_NULL)
@JsonAutoDetect(
    fieldVisibility = JsonAutoDetect.Visibility.ANY,
    getterVisibility = JsonAutoDetect.Visibility.NONE,
    setterVisibility = JsonAutoDetect.Visibility.NONE,
    creatorVisibility = JsonAutoDetect.Visibility.NONE)
public interface JsonTemplate {}
```

Reply:
Note that we do the same thing with layer files (i.e. `addLayerFile`, not `setLayerFiles`)